extensions/blocks/pinterest/pinterest.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
I'm surprised we need this file. Doesn't the Jetpack_Gutenberg class load the assets for the block automatically?

Reply:
We do indeed need it to make the block available and render it.

Reply:
👍🏻